### PR TITLE
6.10 ajax for comments

### PR DESCRIPTION
**Pull request by nelantone**

### CODE REVIEW

File: app/views/products/_comments.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
       <% end %>
       <% unless current_user.nil? %>
         <% if signed_in? && (current_user.admin? || current_user.moderator?) %>
-          <%=  render partial:'comment' %>
+          <%=  render partial:"comments/comment", locals: { comment: comment } %>
```
Comment (nelantone) [inline app/views/products/_comments.html.erb:13]: Why do we need to use locals on Heroku? 
It's funny that I need to use locals remotely and on local you don't need 'locals: {}' to make it work ^^

Comment (wazery) [reply app/views/products/_comments.html.erb:13]: Hmm, that's weird for me

Comment (nelantone) [reply app/views/products/_comments.html.erb:13]: Well... No idea why it didn't work without locals on Heroku :s !

But it's working ^^.

I will take a look later and I will try to figure out :)

Programmers dilema of life

**Comment by nelantone** [comment]
Ok, I have conflics. There is no reason to have it. As is already merged in local and master and there were no conflicts. I'm going to push on master form local/console.
:ok:

**Pull request closed without merging**